res/values/strings_ext.xml

```diff
@@ -138,4 +138,6 @@ Native code debugging (ptrace) slightly weakens the app sandbox. Some apps (most
 
     <string name="view_logs">View logs</string>
 
+    <string name="bluetooth_allow_input_device">Allow input from this device</string>
```

Comment:
```suggestion
    <string name="bluetooth_allow_input_device">Allow this device to act as a keyboard, mouse, or tablet</string>
```

This is more clear as to the effect of this option.  “Input” could refer to _audio_ input.